instrumentation/apache-httpclient/apache-httpclient-4.3/library/src/main/java/io/opentelemetry/instrumentation/apachehttpclient/v4_3/internal/Experimental.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.apachehttpclient.v4_3.internal;
+
+import static java.util.logging.Level.FINE;
+
+import io.opentelemetry.instrumentation.apachehttpclient.v4_3.ApacheHttpClientTelemetryBuilder;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.logging.Logger;
+import javax.annotation.Nullable;
+
+/**
+ * This class is internal and is hence not for public use. Its APIs are unstable and can change at
+ * any time.
+ */
+// TODO (trask) update the above javadoc similar to
+//  https://github.com/open-telemetry/opentelemetry-java/pull/6886
+public class Experimental {
+
+  private static final Logger logger = Logger.getLogger(Experimental.class.getName());
+
+  @Nullable
+  private static final Method emitExperimentalHttpClientMetricsMethod =
+      getEmitExperimentalHttpClientMetricsMethod();
+
+  public void setEmitExperimentalHttpClientMetrics(
```

Comment:
thoughts about the class name and method name?

maybe the method name should be `*Telemetry` instead of `*Metrics`?